docs/release-notes/release-notes-0.16.0.md

```diff
@@ -90,6 +90,9 @@ minimum version needed to build the project.
   so that the user can specify fees during channel creation time in addition
   to the default configuration.
 
+* [Enrich the forwarding](https://github.com/lightningnetwork/lnd/pull/7001) 
```

Comment:
I think this should instead go into the "RPC server" section, since this can also be used by UIs for performance improvements. So it should probably mention both the `ForwardingHistory` RPC and the `lncli fwdinghistory` command.